src/Beacon.jsx

```diff
@@ -13,6 +13,11 @@ const TARGET_CLONE_ID = 'tour-target-clone';
 // If margins are within tolerance then we center tooltip
 const TOOLTIP_TOLERANCE = 50;
 
+const HASH_CHECK_PENDING = 'PENDING';
+const HASH_CHECK_NOT_PERSISTENT = 'NOT_PERSISTENT';
+const HASH_CHECK_FOUND = 'FOUND';
+const HASH_CHECK_NOT_FOUND = 'NOT_FOUND';
```

Comment:
Decided the way it was before was confusing and we should explicitly define all the possible states wrt the hash used for persistence.